sofa-ark-parent/core/common/src/main/java/com/alipay/sofa/ark/common/util/FileUtils.java

```diff
@@ -94,4 +93,16 @@ public static void copyInputStreamToFile(final InputStream source, final File de
                                                                                               throws IOException {
         org.apache.commons.io.FileUtils.copyInputStreamToFile(source, destination);
     }
+
+    /**
+     *
+     * @param path
+     * @return
+     */
+    public static String getCompatiblePath(String path) {
+        if (System.getProperty("os.name").toLowerCase().indexOf("window") > -1) {
+            return path.replace("\\", "/");
+        }
+        return path;
```

Comment:
Seems can directly use `File.seperator` ?

Reply:
> Seems can directly use `File.seperator` ?

yes, but the test case cannot mock windows env, and then the test case will be get fail result.